migrations/alembic.ini

```diff
@@ -43,6 +43,3 @@ formatter = generic
 [formatter_generic]
 format = %(levelname)-5.5s [%(name)s] %(message)s
 datefmt = %H:%M:%S
-
```

Comment:
I added this for a reason.  The scheduler may use the database to do some work, but I don't want it getting processed by alembic.  Why was this removed?